### DISCUSSION TITLE
DRG Case Rates

### DISCUSSION THREAD

**Discussion by narenlulla** [CMSgov/price-transparency-guide#465] [Ideas]
DRG Case Rates are one of the most common types of contracted rate.
When a claim/bill is for a certain DRG (or range of DRGs) the rate is for all services on the bill, the contract does not specify a list of codes that are included.

There can be carve outs, so the contract can say that Specialty Drugs, Implants, Devices etc. are not included ion the rates, again these are not specified by code can usually be translated to ranges of codes.

There does not seem to be any way to represent this most common contract form in the Schema documented

**Comment by shaselton-usds** [comment]
@narenlulla Thanks for reaching out. I'm trying to understand this a bit better; is the question that the schema doesn't have a concept of reporting ranges vs individual items/services?

**Comment by narenlulla** [comment]
The range issue is a different one I have opened.

The Place of Service is a 2 digit code with 60+ values, and 99% of all contracts don't have a different rate based on the place where the service is performed.

In a few cases, e.g. telehealth, a POS of 10 (at home) may have a different rate than POS 11 (in office), though most contracts I have seen don't have a different rate even for those.

Now, since most contracts don't have a POS dependent rate and are silent about POS, many plans that I deal with are putting all the possible POS codes in an array, so 60+ codes in an array for every CPT code in a contract which is unnecessary. We should either come up with a POS-ALL like the CTS-ALL/CTM-00 codes, or make the POS field for professional claims mandatory only if the contracted rate has different rate for the same CPT code and different POS.



Hope that explains the issue.
________________________________
